core/refinerycms-core.gemspec

```diff
@@ -2,7 +2,7 @@
 require File.expand_path('../../core/lib/refinery/version', __FILE__)
 
 version = Refinery::Version.to_s
-rails_version = ['>= 4.2.3', '< 5.0']
+rails_version = ['>= 5.0.0', '< 5.1']
```

Comment:
If you want to do the work to get Appraisals working then by all means give it a go!